sources/procfs.go

```diff
@@ -455,7 +409,14 @@ func parseReadWriteBytes(statsFile string, helpText string, promName string) (me
 	if err != nil {
 		return nil, err
 	}
-	metricList = append(metricList, lustreStatsMetric{title: promName, help: helpText, value: result})
+	l := lustreStatsMetric{
+		title:           promName,
+		help:            helpText,
+		value:           result,
+		extraLabel:      "operation",
```

Comment:
Just curious, is `operation` ever used or displayed anywhere? I know you decide how to handle stats metrics based on the `extraLabelValue` which is empty in this case, but any reason you are setting the `extraLabel`?